dart/2.Dart_basics/Excercises/5_solution.md

```diff
@@ -0,0 +1,8 @@
+# Solution 5
+
+```dart
+void main() {
+  String text = ('This is a newline \nand this is a tab\t and this is a backlash \\');
```

Comment:
Great but update the original course (spanish and english) to introduce the ones that are not mentioned ;-)

Reply:
The `\`to scape is introduced, the only new thing here is the `\t` to make a tabulation. I could not find others to add -.-. Anyway i can add that one in the proper article

Reply:
Yeah! Thanks!